docker-for-aws/faqs.md

```diff
@@ -60,15 +61,54 @@ There are a few work arounds that you can try to get Docker for AWS up and runni
 
 ## Can I use my existing VPC?
 
-Not at this time, but it is on our roadmap for future releases.
+Yes, see [installing Docker for AWS with a pre-existing VPC](index.md#install-with-an-existing-vpc) for more info.
```

Comment:
The title is "Install" not "Installing". We don't use gerunds in titles if we can help it.